class AlreadyExistsError(Exception):
    "Raised when the user already exists in the database"
    pass